src/python/pants/engine/subsystem/native_engine_version

```diff
@@ -1 +1 @@
-e5b11346a458240934cc6136e40faa62b75f79ff
+da39a3ee5e6b4b0d3255bfef95601890afd80709
```

Comment:
After I nuked ~/.cache/bin/native_engine and run a v2 engine task, the version is correct now.